### PR TITLE
feat/last update oss projects

### PR DESCRIPTION
**Pull request by hanakoori01**
### GH Issue

#415

### Steps to test
1. Correr `yarn build`
2. Correr `yarn start`
3. Ir a `localhost:3000`
4. Ir a Pautas para Desarrolladores > Proyectos de Código Abierto

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import React,{ useState, useEffect } from 'react';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import React from 'react';
```
Comment (leisterfrancisco) [inline src/pages/index.js:1]: Is it needed?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — APPROVED** [review]
LGTM

File: src/pages/index.js (hunk 1 above)
Comment (hanakoori01) [reply src/pages/index.js:1]: Yes it is, when i remove it i get this error ReferenceError: React is not defined

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**